src/Symfony/Component/DependencyInjection/Compiler/AbstractRecursivePass.php

```diff
@@ -67,7 +67,6 @@ protected function processValue($value, $isRoot = false)
             $value->setArguments($this->processValue($value->getArguments()));
             $value->setProperties($this->processValue($value->getProperties()));
             $value->setMethodCalls($this->processValue($value->getMethodCalls()));
-            $value->setBindings($this->processValue($value->getBindings()));
```

Comment:
can we have a test case for this? for example within the `RemoveUnusedDefinitionsPassTest`?

Reply:
Sorry I mean the pass that checks if services exist ;p

Reply:
Test added